### ISSUE TITLE
Screen reader compatibility of the CAPTCHA field

### ISSUE THREAD

**Issue by Michael1972** [contao/core#8709]
the frontend for editing formfields is quite accessible for screenreader-users. But the field to resolve captchas could be more logical. In the tab-order of a keyboard-user,, the edit-field to accomplish the addition-task comes first. The task itself is beneath this field. From an accessibility point of view the other way around would be more logical. The task first, beneath it the edit-field. So a screenreader-user by moving down with the cursor-keys would here a correct order.

**Comment by leofeyer** [comment]
Is there a way to switch the tab order of just those two fields? @contao/developers /cc

**Comment by ausi** [comment]
I don’t think there is a robust way to switch the tab order, also because the question is not focusable. Maybe the best way would be to add an `aria-describedby` attribute on the input field that links to the ID of the question element.

**Comment by NinaG** [comment]
Hier ein Beispiel, wie man das macht:
https://www.paciellogroup.com/blog/2014/12/using-aria-describedby-to-provide-helpful-form-hints/

**Comment by leofeyer** [comment]
Can you confirm that the changes in contao/core@490be9e20f8b7f13f88d7b3d4b5b395f188a3a0c are correct?

**Comment by NinaG** [comment]
Seems perfectly correct.

**Comment by ausi** [comment]
I think the ID should depend on `$this->id` as the ID of the input fields itself does, like `captcha_text_<?= $this->id ?>` or `ctrl_<?= $this->id ?>_text`. Otherwise we could end up with duplicate IDs.

**Comment by leofeyer** [comment]
Changed in 02dda3229f8b06c0496a7b4f2ba2880550d10d47.
